demo/traffic_prediction/gen_result.py

```diff
@@ -0,0 +1,47 @@
+res = []
```

Comment:
1. Do not submit pyc file. Use git rebase to purge it.
2. The license of this file is missing.
3. Please give a comments.